### PR TITLE
vuln(NSWG-ECO-388): https-proxy-agent

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

**Comment by lirantal** [comment]
@MarcinHoppe, @ChALkeR, does one of you want to tweet this also?

File: vuln/npm/400.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "id": 400,
+  "title": "Denial of Service",
+  "overview": "`https-proxy-agent` passes unsanitized options to Buffer(arg), resulting in DoS and uninitialized memory leak",
+  "created_at": "2018-02-25",
+  "updated_at": "2018-04-02",
+  "publish_date": "2018-04-02",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "https-proxy-agent",
+  "cves": [
+    ""
```
Comment (ChALkeR) [inline vuln/npm/400.json]: This fails lint. Please use an empty array instead: `[]`.
Also, previous entries where this was missed need fixing.

Comment (lirantal) [reply vuln/npm/400.json]: thanks, good catch

File: vuln/npm/388.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "vulnerable_versions": "<=2.1.1",
+  "patched_versions": ">=2.2.0",
+  "slug": "https-proxy-agent-denial-of-service",
+  "recommendation": "update https-proxy-agent to 2.2.0 or higher",
+  "references": "- https://hackerone.com/reports/319532\n- https://github.com/TooTallNate/node-https-proxy-agent/blob/2.1.1/index.js#L207",
+  "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:H",
+  "cvss_score": 8.2,
+  "coordinating_vendor": ""
+}
+
```
Comment (ChALkeR) [inline vuln/npm/388.json]: Nit: extra newline at the end of file.

**Pull request merged**